CMakeLists.txt

```diff
@@ -84,6 +84,23 @@ set(THIRD_PARTY_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/third_party)
 
 enable_testing()
 
+# # it seems like these need to be at the top level to be resolved by "examples"
```

Comment:
What happens if you uncomment these and remove them from exmaples/CMakeList.txt?